csharp/ExceptionInfo.cs

```diff
@@ -10,6 +10,7 @@ namespace ParquetSharp
     internal sealed class ExceptionInfo
     {
         public delegate IntPtr GetAction<TValue>(out TValue value);
+        public delegate IntPtr GetAction<TValue, TArg0>(TArg0 arg0, out TValue value);
```

Comment:
Swap TArg0 and TValue to be consistent with GetFunction<in TArg0, TValue> below (also notice the in keyword)?

Reply:
Swapped.